tests/integration/services/user.test.ts

```diff
@@ -442,6 +442,59 @@ describe('user service', () => {
             const userUpdated = await UserService.edit(address, data);
 
             expect(userUpdated).to.include(data);
+        })
+    });
+
+    describe('notifications', () => {
+        let users: User[];
+
+        before(async () => {
+            users = await UserFactory({ n: 2 });
+            const yesterday = new Date();
+            yesterday.setDate(yesterday.getDate() - 1);
+            await models.appNotification.bulkCreate([
+                {
+                    address: users[0].address,
+                    type: 1,
+                    params: {
+                        communityId: 1
```

Comment:
I think I see where your idea for the JSON comes from. So you were thinking about objects rather than a string, so it could host more than one param. I honestly think, there will be only one param.

Reply:
I see, in this case, I can change to string

Reply:
changed to string